### ISSUE TITLE
[FEATURE] Get bookmarks from browser itself

### ISSUE THREAD

**Issue by yashchaudhari008** [yashchaudhari008/minime#44]
Find a way to get existing bookmarks of users and show them into app

### Reference:
- https://github.com/annndruha/PulchraBookmarks

**Comment by Prachirsh** [comment]
I would like to work on this project. I am interested to contribute for this feature. Can you please assign it to me?

**Comment by meetbhalodi11** [comment]
@yashchaudhari008 Yes, we can. Chrome give access to local tree and we can fetch bookmarks from there. I can work on this
